app/navigation/router.js

```diff
@@ -92,13 +105,49 @@ class Router extends React.Component {
     };
 
     render = () => {
+        const {
+            leftDrawerOpen,
+            leftDrawerRoute,
+            rightDrawerOpen,
+            rightDrawerRoute
+        } = this.props.navigation;
+
+        let leftDrawerContent;
+        if (leftDrawerRoute) {
+            leftDrawerContent = this.renderRoute(leftDrawerRoute);
+        }
+
+        let rightDrawerContent;
+        if (rightDrawerRoute) {
+            rightDrawerContent = this.renderRoute(rightDrawerRoute);
+        }
+
         return (
-            <NavigationExperimental.Transitioner
-                style={{flex: 1}}
-                navigationState={this.props.navigation}
-                render={this.renderTransition}
-                configureTransition={this.configureTransition}
-            />
+            <Drawer
```

Comment:
Yes, that's how it'll work. We can tweak the props of the drawer to change how much it overlaps